### ISSUE TITLE
App crashes when logging in with authentication token

### ISSUE THREAD

**Issue by sundayLOOKplants** [rigormorrtiss/discordo#332]
`
ws.go:16: Gateway error: websocket: close sent
main.go:18: websocket closed, reason: 4004 Authentication failed.
`

(Token was verified as working using another client.)

**Comment by Cooliosity** [comment]
When I log in with email and password, the app just immediately exits (both windows and arch). I guess this is related

**Comment by ayn2op** [comment]
> `ws.go:16: Gateway error: websocket: close sent main.go:18: websocket closed, reason: 4004 Authentication failed.`

Hello there! I am unable to reproduce this issue, can you please provide reproducible steps?

**Comment by dnet890** [comment]
yes, I got same issue by input my username and password (i have 2fa But, 2fa prompt does not shows up) and I just go back to normal screen

**Comment by ayn2op** [comment]
> yes, I got same issue by input my username and password (i have 2fa But, 2fa prompt does not shows up) and I just go back to normal screen

There is no 2FA prompt, just input the code along with email and password.

**Comment by dnet890** [comment]
My side is working and I don't know about OP problem. I have a question: it will pull latest comments automatically without close and reopen the client right?

**Comment by dnet890** [comment]
Sorry to add something, I tested it on Archlinux WSL1 on Windows 10 and that is correct. I got same problem with OP
